modules/administration/pages/maintenance-window.adoc

```diff
@@ -0,0 +1,17 @@
+[[maintenance-window]]
+= Maintenance Window Checklist
+
+If you work on a system with scheduled maintenance windows, you might find it difficult to remember all the things that you need to during that critical downtime.
+
+This section contains a checklist for your maintenance window, with links to further information on performing each of the steps, if you need it.
+
+
+. Stop services.
```

Comment:
Do we want to single out special services such as "spacewalk services"?  Of course, if you work on a system it is always good if you stop as many services as possible.

Reply:
The original had "SAP and databases", but when I thought about it, it's highly dependent on what you have running. Perhaps a general statement, though?